### DISCUSSION TITLE
在window上可以运行命令行来启动mihomo吗？

### DISCUSSION THREAD

**Discussion by zwl1619** [MetaCubeX/mihomo#1316] [Q&A]
在window上可以运行命令行来启动mihomo吗？要携带配置文件。

**Comment by anaer** [comment]
```
mihomo-windows-amd64.exe -f mihomo.yaml -d .
```

**Comment by jelly21fish** [comment]
我是用powershell运行的，先cd到mihomo所在的文件夹，再输入 .\mihomo -f xxx.yaml

**Comment by apoiston** [comment]
可以写两个批处理管理员运行
mihomo-start.bat   mihomo-stop.bat

@echo off
set "exe=D:\mihomo\mihomo-windows-amd64.exe"
set "config=D:\mihomo\config.yaml"
powershell -WindowStyle Hidden -Command "Start-Process '%exe%' -ArgumentList '-f %config%' -NoNewWindow"



@echo off
set "processName=mihomo-windows-amd64.exe"
set "cache=%USERPROFILE%\.config\mihomo\cache.db"
taskkill /F /IM "%processName%"
del "%cache%"

**Comment by A-caibird** [comment]
命令行运行能实现代理吗， 我是 macos ，使用命令行运行，没有实现网络代理来着

**Comment by c332030** [comment]
开了tun模式，启动了没法上网，相同的配置，debian上可以用

**Reply by A-caibird** [reply]
这个是系统网络缓存问题，有时候你重启系统或者退出就可以了用了，不过不知道是不是 bug
